osu.Game/Screens/Edit/EditorChangeHandler.cs

```diff
@@ -1,31 +1,25 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Text;
 using osu.Framework.Bindables;
 using osu.Framework.Extensions;
-using osu.Game.Beatmaps.Formats;
-using osu.Game.Rulesets.Objects;
 
 namespace osu.Game.Screens.Edit
 {
     /// <summary>
     /// Tracks changes to the <see cref="Editor"/>.
     /// </summary>
-    public partial class EditorChangeHandler : TransactionalCommitComponent, IEditorChangeHandler
+    public abstract partial class EditorChangeHandler : TransactionalCommitComponent, IEditorChangeHandler
```

Comment:
With the way this is being abstracted away further, I'm not sure that the namespace or the xmldoc here make much sense.

Namespace comment also applies to `IEditorChangeHandler` and `TransactionalCommitComponent`.

Reply:
Honestly wasn't sure if we should still keep the underlying `TransactionalCommitComponent`, but left it there to reduce changes for the time being. I'd be okay with moving this to another namespace, but I'm honestly not sure where they should exist. Open to suggestion.

Reply:
Let's leave this where it is until the dust settles and we see the final shape of the skin editor better for now I guess.